signalfx/resource_signalfx_detector.go

```diff
@@ -306,7 +306,7 @@ func getPayloadDetector(d *schema.ResourceData) (*detector.CreateUpdateDetectorR
 	maxDelay := int32(d.Get("max_delay").(int) * 1000)
 	minDelay := int32(d.Get("min_delay").(int) * 1000)
 
-	var tags []string
+	tags := []string{}
```

Comment:
Any chance you can add a test for this?

Reply:
Good point. I've added some scenarios to: resource_signalfx_detector_test